<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>460</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabConnection">
       <attribute name="title">
        <string>Connection</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutConnection">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelUsername">
            <property name="text">
             <string>Username / Custom ID:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditUsername"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayoutMqtt">
          <item row="0" column="0">
           <widget class="QLabel" name="labelHost">
            <property name="text">
             <string>MQTT Host:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditHost">
            <property name="placeholderText">
             <string>e.g. 192.168.1.10</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelPort">
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spinBoxPort">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>1883</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelMqttUser">
            <property name="text">
             <string>MQTT Username:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditMqttUsername"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelMqttPassword">
            <property name="text">
             <string>MQTT Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEditMqttPassword">
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_options">
          <item>
           <widget class="QCheckBox" name="checkBoxPrintOnly">
            <property name="text">
             <string>Print only (ignore actions)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxRemember">
            <property name="text">
             <string>Remember settings</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_connect">
          <item>
           <widget class="QPushButton" name="pushButtonConnect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSaveSettings">
            <property name="text">
             <string>Save Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelTimeout">
            <property name="text">
             <string>Timeout (s):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxTimeoutSec">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelStatus">
            <property name="text">
             <string>Status: Disconnected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditLog">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabActions">
       <attribute name="title">
        <string>Actions</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutActions">
        <item>
         <widget class="QListWidget" name="listWidgetActions"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutActionsButtons">
          <item>
           <widget class="QPushButton" name="pushButtonAddAction">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonEditAction">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDeleteAction">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelActionsInfo">
          <property name="text">
           <string>Expected MQTT message: </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSettings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutSettings">
        <item>
         <widget class="QCheckBox" name="checkBoxStartWithWindows">
          <property name="text">
           <string>Start with Windows</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxAutoConnect">
          <property name="text">
           <string>Connect to MQTT on startup</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxStartMinimized">
          <property name="text">
           <string>Start minimized to tray</string>
          </property>
         </widget>
        </item>
         <item>
          <widget class="QCheckBox" name="checkBoxLockStartupPath">
           <property name="text">
            <string>Lock startup path</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutStartupPath">
           <item>
            <widget class="QLineEdit" name="lineEditStartupPath">
             <property name="placeholderText">
              <string>Select path to mpm.exe...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonBrowseStartupPath">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCreateStartMenuShortcut">
           <property name="text">
            <string>Add to Start Menu</string>
           </property>
          </widget>
         </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
